src/pages/news/2021-06-24-roadmap.md

```diff
@@ -0,0 +1,32 @@
+---
+path: "/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen Juli 2021 - December 2021"
+lang: nl
+template: default
+---
+
+## Roadmap Signalen Juli 2021 - December 2021
+De afgelopen tijd is de product steering groep druk bezig geweest om met alle community leden van Signalen vast te stellen welke verbeteringen en doorontwikkelingen zij graag zouden zien ten aanzien van Signalen.
+
+Het thema is dan ook om ons in de 2e helft van dit jaar te gaan richten op het gebruikersvriedenlijker maken
+van Signalen voor de mensen die de meldingen afhandelen. Hieronder volgt op hoofdlijnen aan welke onderdelen we zullen gaan werken:
+
+---
+
+| Prioriteit | Functie thema (epic) | Beschrijving | Complexiteit/ Omvang van werk |
+| -------- | -------- | -------- | -------- |
+| 1     | Granulariteit van lees- en schrijfrechten | Dropdown aanpassen naar gebruikers die alleen kunnen lezen -> small (afhankelijk van 1) | medium / large    |
+| 2a     | Filtering |  Refactor |  large    |
+| 2b     | Filtering |  Autocomplete toevoegen adres |  small    |
+| 3a     | Gebruikersvriendelijker beheer | Routeringsregels in Django admin |  small    |
+| 3b     | Gebruikersvriendelijker beheer | Machine learning, onderzoek nodig |  large   |
+| 3c     | Gebruikersvriendelijker beheer | Aanvullende vragen implementer in front-end|  large   |
+| 4     | Verrijken data dump | Verrijken data dump|  small/medium  |
+| 4     | Notificeren van behandelaars| Mailtje versturen / Instelbaar maken |  medium  |
+
+---
+<br/>
```

Comment:
Is this inline html really needed? It looks quite good without it to me.

Reply:
yes as the line comes to close to the table which I think makes it not readable